src/EA.Weee.Integration.Tests/Handlers/GetAatfNotesRequestHandlerIntegrationTests.cs

```diff
@@ -253,6 +254,119 @@ public class WhenIGetFilteredEvidencesBasedOnAllowedStatusesList : GetAatfNotesR
+
+                var evidence2 = EvidenceNoteDbSetup.Init()
+                .WithOrganisation(organisation.Id)
+                .WithAatf(aatf.Id)
+                .With(n =>
+                {
+                    n.UpdateStatus(NoteStatusDomain.Submitted, UserId.ToString());
+                    n.UpdateStatus(NoteStatusDomain.Approved, UserId.ToString());
+                })
+                .WithComplianceYear(SystemTime.UtcNow.Year)
+                .Create();
+
+                var evidence3 = EvidenceNoteDbSetup.Init()
+                .WithOrganisation(organisation.Id)
+                .WithAatf(aatf.Id)
+                .With(n =>
+                {
+                    n.UpdateStatus(NoteStatusDomain.Submitted, UserId.ToString());
+                    n.UpdateStatus(NoteStatusDomain.Approved, UserId.ToString());
+                })
+                .WithComplianceYear(SystemTime.UtcNow.Year)
+                .Create();
+
+                var evidence4 = EvidenceNoteDbSetup.Init()
+                .WithOrganisation(organisation.Id)
+                .WithAatf(aatf.Id)
+                .With(n =>
+                {
+                    n.UpdateStatus(NoteStatusDomain.Submitted, UserId.ToString());
+                    n.UpdateStatus(NoteStatusDomain.Approved, UserId.ToString());
+                })
+                .WithComplianceYear(SystemTime.UtcNow.Year)
+                .Create();
+
+                var evidence5 = EvidenceNoteDbSetup.Init()
+                .WithOrganisation(organisation.Id)
+                .WithAatf(aatf.Id)
+                .With(n =>
+                {
+                    n.UpdateStatus(NoteStatusDomain.Submitted, UserId.ToString());
+                    n.UpdateStatus(NoteStatusDomain.Approved, UserId.ToString());
+                })
+                .WithComplianceYear(SystemTime.UtcNow.AddYears(-1).Year)
+                .Create();
+
+                var evidence6 = EvidenceNoteDbSetup.Init()
+                .WithOrganisation(organisation.Id)
+                .WithAatf(aatf.Id)
+                .With(n =>
+                {
+                    n.UpdateStatus(NoteStatusDomain.Submitted, UserId.ToString());
+                    n.UpdateStatus(NoteStatusDomain.Approved, UserId.ToString());
+                })
+                .WithComplianceYear(SystemTime.UtcNow.AddYears(-1).Year)
+                .Create();
+            };
+
+            private readonly Because of = () =>
+            {
+                evidenceNoteData = Task.Run(async () => await handler.HandleAsync(new GetAatfNotesRequest(organisation.Id, aatf.Id, allowedStatuses, null, SystemTime.UtcNow.Year, null, null, null, null, null))).Result;
+            };
+
+            private readonly It shouldReturnAFilteredListBasedOnAllowedStatuses = () =>
+            {
+                evidenceNoteData.Should().NotBeNull();
+            };
+
+            private readonly It shouldHaveExpectedResultsCount = () =>
+            {
```

Comment:
Think we need to update this assert as  currently it just asserts that things that have been added to the list during the test setup are still in the list.

Suggest something like,
evidenceNoteData.Count(n => n.ComplianceYear == SystemTime.UtcNow.AddYears(-1).Year).Should().Be(0);